src/shared/Elements/H1.tsx

```diff
@@ -18,6 +21,9 @@ export const H1 = styled.h1<IH1Props>`
   margin-bottom: ${(props) => props.marginBottom || '18px'};
   margin-top: ${(props) => props.marginTop || 'initial'};
   display: ${(props) => props.display || ''};
+  font-weight: ${(props) => props.fontWeight || 'normal'};
```

Comment:
this breaks UI on login page, but we can fix it later in HCK-148

![image](https://user-images.githubusercontent.com/23108291/70296356-5a9ecf80-17b8-11ea-9784-1a5b029f1ff0.png)